xbmc/threads/platform/pthreads/ThreadImpl.cpp

```diff
@@ -61,6 +61,10 @@ static pid_t GetCurrentThreadPid_()
   return gettid();
 #elif TARGET_DARWIN_TVOS
   return pthread_mach_thread_np(pthread_self());
+#elif defined(__APPLE__) && defined(__MACH__)
```

Comment:
why not using the same as tvOS?

Reply:
Because it was not the the first thing google suggested as a replacement ;)

Reply:
But much cleaner, thanks ;)